README.md

```diff
@@ -50,6 +50,112 @@ When implementing a change:
+
+Building Lotus requires some system dependencies, usually provided by your distribution.
+
+Ubuntu/Debian:
+```
+sudo apt install mesa-opencl-icd ocl-icd-opencl-dev gcc git bzr jq pkg-config curl clang build-essential hwloc libhwloc-dev wget -y && sudo apt upgrade -y
+```
+
+Fedora:
+```
+sudo dnf -y install gcc make git bzr jq pkgconfig mesa-libOpenCL mesa-libOpenCL-devel opencl-headers ocl-icd ocl-icd-devel clang llvm wget hwloc libhwloc-dev
+```
+
+For other distributions you can find the required dependencies [here.](https://docs.filecoin.io/get-started/lotus/installation/#system-specific) For instructions specific to macOS, you can find them [here.](https://docs.filecoin.io/get-started/lotus/installation/#macos)
+
+#### Rustup
+
+Lotus needs [rustup](https://rustup.rs). The easiest way to install it is:
+
+```bash
+wget -c https://golang.org/dl/go1.16.2.linux-amd64.tar.gz -O - | sudo tar -xz -C /usr/local
+```
+
+#### Go
+
+To build Lotus, you need a working installation of [Go 1.16.1 or higher](https://golang.org/dl/):
+
+```bash
+wget -c https://golang.org/dl/go1.16.2.linux-amd64.tar.gz -O - | sudo tar -xz -C /usr/local
+```
+
+**TIP:**
+You'll need to add `/usr/local/go/bin` to your path. For most Linux distributions you can run something like:
+
+```shell
+echo "export PATH=$PATH:/usr/local/go/bin" >> ~/.bashrc && source ~/.bashrc
+```
+
+See the [official Golang installation instructions](https://golang.org/doc/install) if you get stuck.
+
+### Build and install Lotus
+
+Once all the dependencies are installed, you can build and install the Lotus suite (`lotus`, `lotus-miner`, and `lotus-worker`).
+
+1. Clone the repository:
+
+   ```sh
+   git clone https://github.com/filecoin-project/lotus.git
+   cd lotus/
+   ```
+
+2. To join mainnet, checkout the [latest release](https://github.com/filecoin-project/lotus/releases).
+
+   If you are changing networks from a previous Lotus installation or there has been a network reset, read the [Switch networks guide](https://docs.filecoin.io/get-started/lotus/switch-networks/) before proceeding.
+
+   For networks other than mainnet, look up the current branch or tag/commit for the network you want to join in the [Filecoin networks dashboard](https://network.filecoin.io), then build Lotus for your specific network below.
+
+   ```sh
+   git checkout <tag_or_branch>
+   # For example:
+   git checkout <vX.X.X> # tag for a release
+   ```
+
+   Currently, the latest code on the _master_ branch corresponds to mainnet.
+
+3. If you are in China, see "[Lotus: tips when running in China](https://docs.filecoin.io/get-started/lotus/tips-running-in-china/)".
+4. Depending on your CPU model, you will want to export additional environment variables:
+
+   If you have **an AMD Zen or Intel Ice Lake CPU (or later)**, enable the use of SHA extensions by adding these two environment variables:
+
+   ```sh
+   export RUSTFLAGS="-C target-cpu=native -g"
+   export FFI_BUILD_FROM_SOURCE=1
```

Comment:
I'd say that the basic instructions should probably use the prebuilt proof binaries - this way you don't have to install rustup, and build the proofs library which takes a while. Then we can link to the instructions on docs.filecoin.io for people who may want a full from-source build

Reply:
I don’t think the proof prebuilds are compatible with different systems atm? And isn’t building from source a pretty standard thing to do now?

Reply:
oh checked with nemo, i was wrong, magik is right. @rjan90 mind making the update?